src/jsrs_parser.cc

```diff
@@ -89,8 +90,17 @@ Local<Value> Parse(Isolate* isolate, const String::Utf8Value& in) {
   }
 
   size_t parsed_size = 0;
-  Local<Value> result =
-      (kParseFunctions[type])(isolate, str + start_pos, end, &parsed_size);
+  Local<Value> result;
+  {
+    TryCatch trycatch(isolate);
+    result =
+        (kParseFunctions[type])(isolate, str + start_pos, end, &parsed_size);
+
```

Comment:
Just curious, is this space necessary, as you don't have it in the other places?